platformcomponents/android/checkbox.json

```diff
@@ -0,0 +1,27 @@
+{
+  "checkbox": {
+    "figma": "https://www.figma.com/file/z6oi9Ku3X7rM1Wj2451B9u/Components---Android?node-id=591%3A93",
+    "checked":{
+      "#normal":{
+        "background": "@theme-control-active-normal",
+        "text": "@theme-text-primary-normal",
+        "icon": "@theme-text-primary-inverted"
+      },
+      "#disabled":{
+        "background": "@theme-control-active-disabled",
+        "text": "@theme-text-primary-disabled",
+        "icon": "@theme-text-primary-inverted"
+      }
+    },
+    "unchecked":{
+      "#normal":{
+        "border":"@theme-page-control-normal",
+        "text": "@theme-text-primary-normal"
+      },
+      "#disabled":{
+        "border":"@theme-control-inactive-disabled",
```

Comment:
Looking on Figma unchecked/disabled has a background but no border.

Reply:
ah, I see that is my misunderstanding

Reply:
Should be resolved now @paulwitty